examples/default/realign.json

```diff
@@ -0,0 +1,67 @@
+{
+  "@context": "https://github.com/Inria-Visages/blob/master/BIDS-prov/context.json",
+  "@id": "http://example.org/ds00000X",
+  "generatedAt": "2020-03-10T10:00:00",
+  "wasGeneratedBy": {
+      "@id": "INRIA",
+      "@type": "Project",
+      "startedAt": "2016-09-01T10:00:00",
+      "wasAssociatedWith": { "@id": "NIH",
+                              "@type": "Organization",
+                              "hadRole": "Funding"
+                            }
+    },
+    "exp_dir" : {
```

Comment:
I would remove "exp_dir" for now, unless this is in the BIDS-prov spec?

The reason is that this is a very SPM specific attribute and I'm not sure how this would translate for other neuroimaging pipelines.